Gruntfile.js

```diff
@@ -40,7 +70,7 @@ module.exports = function(grunt) {
       },
       relativeRoot: {
         options: {
-          root: "test/fixtures/",
+          pathPrefix: "test/fixtures/",
```

Comment:
I don't really agree with changing this.  I think we should just add documentation around what 'root' is.  pathPrefix doesn't really say what this does either.